docs/app/components/documentation-category/documentation-category.component.ts

```diff
@@ -13,9 +14,24 @@ export class DocumentationCategoryComponent implements OnInit, AfterViewInit {
 
     private category: ICategory;
     private trackScroll: boolean = false;
+
+    angular: boolean = true;
```

Comment:
As mentioned in the class below, remove this and compare directly to version in service